packages/gafl-webapp-service/src/pages/oidc/role-required.njk

```diff
@@ -0,0 +1,23 @@
+{% extends "layout.njk" %}
+{% from "button/macro.njk" import govukButton %}
+{% block pageTitle %}Role required - GOV.UK{% endblock %}
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l">Role required</h1>
+        <p class="govuk-body">
+            You were successfully signed in but you do not have the correct role assigned to access the telesales application.
+        </p>
+        <p class="govuk-body">
+            Please contact your Microsoft Dynamics administrator to request access.
+        </p>
+
+        {{ govukButton({
+            href: '/buy',
```

Comment:
I have pushed the uri's down onto the pages so we can relocate anything easily - no hardcoded uri's.

Reply:
Refactored to use CONTROLLER.uri